h2o-algos/src/main/java/hex/schemas/IsolationForestV3.java

```diff
@@ -0,0 +1,38 @@
+package hex.schemas;
+
+import hex.tree.isofor.IsolationForest;
+import hex.tree.isofor.IsolationForestModel;
+import water.api.API;
+
+public class IsolationForestV3 extends SharedTreeV3<IsolationForest, IsolationForestV3, IsolationForestV3.IsolationForestParametersV3> {
+
+    public static final class IsolationForestParametersV3 extends SharedTreeV3.SharedTreeParametersV3<IsolationForestModel.IsolationForestParameters, IsolationForestParametersV3> {
+        static public String[] fields = new String[]{
+                "model_id",
+                "training_frame",
+                "score_each_iteration",
+                "score_tree_interval",
+                "ignored_columns",
+                "ignore_const_cols",
+                "ntrees",
+                "max_depth",
+                "min_rows",
+                "max_runtime_secs",
+                "seed",
+                "build_tree_one_node",
+                "mtries",
+                "sample_size",
+                "sample_rate",
+                "col_sample_rate_change_per_level",
```

Comment:
What about col_sample_rate?

Reply:
You have good eyes!

Isolation Forest is very similar to DRF, the API here is consistent with DRF in this respect. Both DRF and IsoForest uses `mtries` instead of `col_sample_rate`.